packages/flutter/lib/src/cupertino/search_field.dart

```diff
@@ -274,7 +274,9 @@ class CupertinoSearchTextField extends StatefulWidget {
   /// {@macro flutter.material.textfield.onTap}
   final VoidCallback? onTap;
 
-  /// {@macro flutter.widgets.editableText.autocorrect}
+  /// Whether to enable autocorrection.
+  ///
+  /// Defaults to true.
```

Comment:
CupertinoSearchTextField does not have autofillHints, so default is always true.